test/t8_IO/t8_gtest_vtk_writer.cxx

```diff
@@ -114,6 +116,24 @@ use_c_interface<t8_cmesh_t> (const t8_cmesh_t grid, const char *fileprefix, [[ma
 #endif
 }
 
+void
+vtk_writer_test_fill_data (const t8_locidx_t cells_to_write_count, std::vector<double> &scalar_data,
+                           std::vector<double> &vector_data)
+{
```

Comment:
```suggestion
{
  scalar_data.resize (cells_to_write_count);
  vector_data.resize (3 * cells_to_write_count);
```

I think it is better to use a single allocation instead of `cells_to_write_count` many

Reply:
I do not think it makes a real difference due to the intelligent handling of memory of std::vector (i.e. not allocation one item at a time).